crates/bevy_pbr/src/material.rs

```diff
@@ -794,7 +794,18 @@ pub struct PreparedMaterial<T: Material> {
 }
 
 #[derive(Component, Clone, Copy, Default, PartialEq, Eq, Deref, DerefMut)]
-pub struct MaterialBindGroupId(Option<BindGroupId>);
+pub struct MaterialBindGroupId(pub Option<BindGroupId>);
```

Comment:
```suggestion
pub struct MaterialBindGroupId(pub Option<BindGroupId>);

```

Style nit: space between blocks.